test-packages/openui5-snapshot-test/output-dts/sap.ui.core.d.ts

```diff
@@ -15148,8 +15148,10 @@ declare namespace sap {
             }
           ): Promise<any>;
         }
+        export const RuleEngineOpaAssertions: RuleEngineOpaAssertions;
 
         interface usage {}
+        export const usage: usage;
```

Comment:
BTW: @codeworrior wants to get rid of the empty namespaces which is used to define the types. In future, we even might get rid of this.

Reply:
sounds like a plan :)